operators/src/processing/reprojection.rs

```diff
@@ -230,18 +280,22 @@ where
     ) -> Result<BoxStream<'a, Result<Self::Output>>> {
         let rewritten_query = reproject_query(query, self.from, self.to)?;
 
-        Ok(self
-            .source
-            .query(rewritten_query, ctx)
-            .await?
-            .map(move |collection_result| {
-                collection_result.and_then(|collection| {
-                    CoordinateProjector::from_known_srs(self.from, self.to)
-                        .and_then(|projector| collection.reproject(projector.as_ref()))
-                        .map_err(Into::into)
+        if let Some(rewritten_query) = rewritten_query {
+            Ok(self
+                .source
+                .query(rewritten_query, ctx)
+                .await?
+                .map(move |collection_result| {
+                    collection_result.and_then(|collection| {
+                        CoordinateProjector::from_known_srs(self.from, self.to)
+                            .and_then(|projector| collection.reproject(projector.as_ref()))
+                            .map_err(Into::into)
+                    })
                 })
-            })
-            .boxed())
+                .boxed())
+        } else {
+            Ok(Box::pin(stream::empty())) // TODO: should be empty collection?
```

Comment:
probably, but maybe an error?

when does this happen?

Reply:
this will also return none if the query bounds do not intersect with the area of use of one of the projections. I don't know if this should return an error. When you query with a different bbox (that overlaps the data) it will work. In the raster case this happens all the time when wms tiles are requested that are not inside the SRS area of use.

Reply:
Maybe add a test for that and then it will be more clear, what we expect?

Reply:
[607f82c](https://github.com/geo-engine/geoengine/pull/656/commits/607f82c1a71f4d1cf3d6897f9766e3564316bfd2)